sources/packages/web/src/views/aest/institution/ViewOfferingChangeRequest.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <full-page-container layout-template="centred-tab">
+    <template #header>
+      <header-navigator
+        title="Study period offerings"
+        :routeLocation="{ name: AESTRoutesConst.OFFERING_CHANGE_REQUESTS }"
+        subTitle="View Request"
+      />
+      <program-offering-detail-header
+        class="m-4"
+        :headerDetails="headerDetails"
+      />
+    </template>
+    <template #alerts>
+      <banner class="mb-2" :type="BannerTypes.Warning" :header="bannerText" />
+    </template>
+    <template #tab-header>
+      <v-tabs v-model="tab" color="primary">
+        <v-tab value="requested-change" :ripple="false">Requested Change</v-tab>
+        <v-tab value="active-offering" :ripple="false">Active Offering</v-tab>
+      </v-tabs>
+    </template>
+    <v-window v-model="tab">
+      <v-window-item value="requested-change"
+        ><offering-change-request
+          v-if="tab === 'requested-change'"
```

Comment:
`v-if` is not supposed to be needed here, right?
It is a Vuetify beta issue?

Reply:
Kind of. I couldn't find any other way to load v-tab on demand.

Reply:
v-if removed and going by v-tabs default behaviour where eager=false for v-window-item